pkgs/tools/graphics/timg/default.nix

```diff
@@ -1,26 +1,28 @@
-{ lib
-, stdenv
+{ cmake
 , fetchFromGitHub
-, cmake
-, pkg-config
 , ffmpeg
 , graphicsmagick
+, lib
 , libdeflate
 , libexif
 , libjpeg
 , libsixel
 , openslide
+, pkg-config
+, stb
+, qoi
+, stdenv
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
Oh nice ! Thank you ! I don't feel alone to do that anymore!

Reply:
Yes, it was you pointing me to it in my [last PR](https://github.com/NixOS/nixpkgs/pull/245990) :)